modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/TypeScriptAngular2ClientCodegen.java

```diff
@@ -26,6 +26,7 @@
     public static final String NPM_VERSION = "npmVersion";
```

Comment:
As you may know, this file is conflicting my previous [commit](https://github.com/swagger-api/swagger-codegen/pull/5569).